.github/workflows/pytest.yml

```
@@ -0,0 +1,35 @@
+
+      - name: Install FFmpeg (Linux)
+        run: sudo apt-get update && sudo apt-get install -y ffmpeg
+    
+      - name: Install dependencies
+        run: uv sync
+
+      - name: Run Tests with Pytest
+        run: pytest --cov=app --cov-report=xml
+  
+      - name: Upload Coverage Report
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)